### DISCUSSION TITLE
More frequent releases?

### DISCUSSION THREAD

**Discussion by alensiljak** [extrawurst/gitui#1581] [General]
Just wondering if more frequent releases would be useful to massive but less frequent ones. 
I'd like, for example, to be able to use fetch on the phone. That feature seems ready but it is not very convenient to get it on the device.

**Comment by extrawurst** [comment]
what is keeping you from building from source?

**Reply by alensiljak** [reply]
I'm not entirely sure how to build a binary that will run in Termux. 
I tried at some point but there was a mess with the destination platform if building on Windows or Linux. And building on Android itself is not supported.
